# Copyright 2025 Reazon Holdings, Inc.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

LeaderNode:
  ros__parameters:
    Dn: [0.700,    0.700,   0.700,   0.700,   0.120,   0.110,  0.150,  0.050]
    Jn: [0.550,    0.350,   0.100,   0.018,   0.016,   0.010,  0.008,  0.014]
    gn: [20.000,   20.000,  18.000,  24.000,  15.000,  14.000, 14.000, 10.000]
    Kp: [1000.000, 1100.000, 5250.000, 5300.000, 1000.000, 500.000, 300.000, 350.000]
    Kd: [17.000,   18.000,  90.000,  94.000,  60.000,  20.000, 20.000, 20.000]
    Kf: [2.600,    2.600,   1.800,   2.800,   0.800,   0.800,  1.00,  0.000]
    Fc: [0.336,    0.336,   0.450,   0.146,   0.070,   0.093,  0.212,  0.212]
    k:  [28.417,  28.417,  29.065,  130.038, 151.771, 242.287, 7.888,  7.000]
    Fv: [0.063,    0.0630,   0.604,   0.813,   0.029,   0.072,  0.084,  0.084]
    Fo: [0.088,    0.088,   0.008,  -0.058,   0.005,   0.009, -0.059, -0.050]

